src/com/twu/biblioteca/MainMenu.java

```diff
@@ -1,40 +1,62 @@
 package com.twu.biblioteca;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Scanner;
 
 public class MainMenu {
 
-    public void welcome() {
-        System.out.println("Welcome to Biblioteca. Your one-stop-shop for great book titles in Bangalore!");
+    private final String welcome = "Welcome to Biblioteca. Your one-stop-shop for great book titles in Bangalore!";
+    private final String invalidMessage = "Please select a valid option!";
+    private final String goodBye = "Goodbye!";
+
+   ArrayList<String> books = new ArrayList<String>(Arrays.asList(
+           "1. Clean Code | Robert C. Martin | 2008",
+           "2. Don't Make Me Think | Steve Krug | 2000",
+           "3. Test Driven Development | Kent Beck | 2000"));
+
+   ArrayList<String> options = new ArrayList<String>(Arrays.asList(
```

Comment:
If you don't make something private or public in Java, what sort of access is allowed?  Is that what you want here?

Reply:
It is visible to all classes in the same package. I should make it private as it is only being used by the class it is within. Normally, getters and setters generally would be expected to be public.

Reply:
That's a good approach: private until proven otherwise.